Admin/AdminInterface.php

```diff
@@ -656,6 +657,21 @@ public function delete($object);
     /**
      * @param mixed $object
      *
+     * @return int|null
+     */
+    public function getVersion($object);
```

Comment:
Actually there is no need for this as the ``LockInterface`` interface already exist in this PR.